### ISSUE TITLE
Getting same probability of labels in all the test set examples

### ISSUE THREAD

**Issue by Rotramartya** [google-research/bert#688]
The number of labels is 32. It gives 0.25 probability of label '0' in all the test set examples. Similarly, it gives 0.017 probability for label '1' in the test set examples. And so on. Please help

**Comment by sbmark** [comment]
I'm going through the same thing. 
Input : one sentence (text_a)  // Number of label is 1847.
all test probabilities are similar  and  top label is same. ( top label changes for each model )
Question ) Can't bert handle many labels ? ( Can bert handle about 2 or 3 labels ? )

probabilities and argmax label log
INFO:tensorflow:{'probabilities': array([5.9799670e-04, 2.2309359e-04, 6.0582173e-04, ..., 6.4004096e-05,
       6.5070941e-05, 5.2951513e-05], dtype=float32), 'label_position': 855}

Please reply, thank you

**Comment by Rotramartya** [comment]
@sbmark BERT can handle multiclass classification (ref: https://www.kaggle.com/thebrownviking20/bert-multiclass-classification ). Multiclass is not the issue, I guess. Someone suggested me that model is not trained properly, as I am training the model with just 1 epoch and the learning rate is 0.01. So, I am trying to train the model again with improved hyperparameters. 

By the way, how many epochs did you use? That might be an issue. If not, please reply if you find a solution.

Thanks.

**Comment by Rotramartya** [comment]
Well, the problem is the model is not getting trained properly. I changed the hyperparameters which solved the issue.
